packages/pipeline-editor/src/PipelineExportDialog.tsx

```diff
@@ -74,6 +74,10 @@ export class PipelineExportDialog extends React.Component<IProps, IState> {
     if (!platformSelection) {
       return new Array<Record<string, string>>();
     } else if (platformSelection === KFP_SCHEMA) {
+      // TODO: remove temporary workaround for KFP Python DSL export option
```

Comment:
Could we augment this with a link to the issue to provide context information? As is, it's not clear what prompted the TODO and a developer would have to inspect the file history to determine why the temporary change was made.

Reply:
That's a good idea. I'll add that now